service/src/main/java/com/codeforcommunity/processor/BlocksProcessorImpl.java

```diff
@@ -188,12 +187,25 @@ public void seedBlockCompletions(JWTData jwtData, List<BlockSeedingInfo> blockSe
 
     blockSeedingInfos.forEach(
         (bsi) -> {
+          String original =
+              db.selectFrom(BLOCK).where(BLOCK.ID.eq(bsi.getId())).fetchOne().toString();
+
+          AuditRecord audit = db.newRecord(AUDIT);
+          audit.setTableName("block");
+          audit.setTransactionType("update");
+          audit.setOldValue(original);
+
           db.update(BLOCK)
               .set(BLOCK.STATUS, BlockStatus.DONE)
               .set(BLOCK.LAST_COMPLETED, bsi.getDateCompleted())
               .setNull(BLOCK.ASSIGNED_TO)
               .where(BLOCK.ID.eq(bsi.getId()))
               .execute();
+
+          audit.setResult(original);
```

Comment:
We're setting oldValue and result to be the same thing?

Reply:
hmm this is setting Result after UPDATE is called so it would be the updated record then ? I will double check that this works out